Binary files a/src/platformsupport/fontdatabases/mac/qfontengine_coretext.mm and b/src/platformsupport/fontdatabases/mac/qfontengine_coretext.mm differ
diff -u -r --label a/src/platformsupport/fontdatabases/mac/qfontengine_coretext.mm --label b/src/platformsupport/fontdatabases/mac/qfontengine_coretext.mm a/src/platformsupport/fontdatabases/mac/qfontengine_coretext.mm b/src/platformsupport/fontdatabases/mac/qfontengine_coretext.mm
--- a/src/platformsupport/fontdatabases/mac/qfontengine_coretext.mm
+++ b/src/platformsupport/fontdatabases/mac/qfontengine_coretext.mm
@@ -764,7 +764,7 @@
 
 QFixed QCoreTextFontEngine::emSquareSize() const
 {
-    return QFixed::QFixed(int(CTFontGetUnitsPerEm(ctfont)));
+    return QFixed(int(CTFontGetUnitsPerEm(ctfont)));
 }
 
 QFontEngine *QCoreTextFontEngine::cloneWithSize(qreal pixelSize) const
